hr_employee_customers_tab/__manifest__.py

```diff
@@ -15,6 +15,6 @@
         'views/hr_views.xml',
     ],
     "application": False,
-    "installable": True,
+    "installable": False,
```

Comment:
If this module is phased out, add the "Depricated" keyword to title

Reply:
Because this module can be useful sometime in the future and is not replaced by `hr_outplacement_tab` or an other module.